### PR TITLE
Adding functions `st_asgeojson` and `st_geomfromgeojson`

### PR DESCRIPTION
**Pull request by JCOR11599**

### CODE REVIEW

**Review by andrew-wm-arthur — COMMENTED** [review]
Need to add enginetests and round-trip tests showing you can go back an forth from the JSON representation accurately

File: sql/expression/function/geojson.go [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+	// Evaluate precision
+	p, err := g.ChildExpressions[1].Eval(ctx, row)
+	if err != nil {
+		return nil, err
+	}
+	if p == nil {
+		return nil, nil
+	}
+	// Must be an int type
+	_p := 0
```
Comment (andrew-wm-arthur) [inline sql/expression/function/geojson.go:167]: what's up with these underscore variable names?

**Review by andrew-wm-arthur — APPROVED** [review]
LGTM!

File: sql/expression/function/geojson.go (hunk 1 above)
Comment (JCOR11599) [reply sql/expression/function/geojson.go:167]: Just a habit from Python

**Pull request merged**